frontend/components/hamburger/component.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FC, ReactNode } from 'react';
+import cx from 'classnames';
+
+export interface HamburgerProps {
+  isOpen: boolean;
+  color: 'white' | 'black';
+  onClick?: (event: React.MouseEvent<HTMLButtonElement>) => void;
+  className?: string;
+  children?: ReactNode;
+}
+
+const COLOR = {
+  white: 'text-white',
+  black: 'text-black',
+};
+
+const Hamburger: FC<HamburgerProps> = ({ color = 'white', className, isOpen, onClick }: HamburgerProps) => {
+  const barCSS = 'block absolute h-0.5 w-5 bg-current transform transition duration-500 ease-in-out';
+
+  return (
+    <button
+      className={cx('w-10 h-10 relative focus:outline-none', className, { [COLOR[color]]: true })}
+      onClick={onClick}
+    >
+      <div className="block w-5 absolute left-1/2 top-1/2 transform -translate-x-1/2 -translate-y-1/2">
```

Comment:
Could you add roles (menu, menuitem) and aria labels (expanded, labelledby, ...)? Please check this article: https://www.smashingmagazine.com/2017/11/building-accessible-menu-systems/